lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reverification.yaml

```diff
@@ -1,23 +1,189 @@
-name: Reverification
-
+name: MFA Reset
 description: >-
-  The routes a user can take to revalidate their
-  account. They will prove their identity but we
-  won't store it.
+  The routes a user can take to validate their
+  account so that they can reset their two
+  factor authentication mechanism.
 
 states:
   # Entry points
-
   START:
     events:
       next:
-        # should be filled out in https://govukverify.atlassian.net/browse/PYIC-6486
-        targetState: PLACEHOLDER
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR_NO_TICF
+        checkFeatureFlag:
+          ticfCriBeta:
```

Comment:
This was intentional as the flow requires ticf & I believe you can only get onto this journey if the mfaReset feature flag is set

Reply:
What do you mean by 'requires TICF', do you mean we shouldn't support this journey if TICF is unavailable?

Reply:
It looked like it was important in the MURAL design. Since I don't know when each is getting rolled out I thought that implied it was a dependency but I don't know the ordering of roll-outs so happy to make this flow inclusive

Reply:
Yeah, I would treat it like the TICF CRI in all the other places.

I think it's meant to be enabled soon(TM) which will make this easier to manage!